app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>SampleApp</title>
+    <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
```

Comment:
sau đó áp dụng vào các text của TOÀN BỘ file HTML nhé
Ruby on Rails Tutorial Sample App